content/en/project/MEG-effects-of-different-attributes-of-words-in-word-processing/index.md

```diff
@@ -0,0 +1,121 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2023-06-07" # Date you first upload your project.
+# Title of your project (we like creative title)
+title: "The Effects of Word Frequency, Orthographic Neighborhood and Part of Speech in Word Processing: A MEG Study"
+
+# List the names of the collaborators within the [ ]. If alone, simple put your name within []
+names: [Ruo-Chi Yao]
+
+# Your project GitHub repository URL
+github_repo: https://github.com/Ruo-Chi/Ruo-Chi-Yao_project
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [meg, github, mne, python]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "Word processing is influenced by different attributes of words, which can be observed vis brain imaging techniques such as MEG. Let's find evidence of the influence of word attributes by analyzing a dataset."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: ""
```

Comment:
Please add a cover image for your project.

Reply:
I have added a cover image. Thank you so much.